pom.xml

```diff
@@ -241,6 +241,23 @@
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>properties-maven-plugin</artifactId>
```

Comment:
I'm not sure about having it run all the time, even outside of the cloud build, but you might want to put a comment here describing its current purpose anyway.